src/main/lrsql/system/webserver.clj

```diff
@@ -101,14 +109,13 @@
                :enable-reaction-routes    enable-reactions
                :oidc-interceptors         oidc-admin-interceptors
                :oidc-ui-interceptors      oidc-admin-ui-interceptors
-               :head-opts
-               {:sec-head-hsts         sec-head-hsts
-                :sec-head-frame        sec-head-frame
-                :sec-head-content-type sec-head-content-type
-                :sec-head-xss          sec-head-xss
-                :sec-head-download     sec-head-download
-                :sec-head-cross-domain sec-head-cross-domain
-                :sec-head-content      sec-head-content}}))
+               :head-opts head-opts}))
+        routes (add-openapi-route
```

Comment:
this is just opinion but it seems weird to me to apply this to the finished routes not just add it in the same clause that adds admin routes.

Reply:
got it